docs/pages/admin-guides/management/operations/backup-restore.mdx

```diff
@@ -219,23 +126,7 @@ also apply to a new cluster being bootstrapped from the state of an old cluster:
   - Some user authentication mechanisms (e.g. WebAuthn) require that the public
     endpoint of the Web UI remains the same. This cannot be caught by Teleport,
     so be careful!
-  - Any Node whose invite token is defined in the Auth Service's configuration
-    file will be able to join automatically, but Nodes that were added
-    dynamically will need to be re-invited.
-
-</TabItem>
-<TabItem label="Teleport Enterprise Cloud">
-
-In Teleport Enterprise Cloud, backend data is managed for you automatically. 
-
-If you would like to migrate configuration resources to a self-hosted Teleport
-cluster, follow our recommended backup practice of storing configuration
-resources in a git repository and running `tctl create -f` regularly for each
-resource. 
-
-This will enable you to keep your configuration resources up to date regardless
-of storage backend.
-
-</TabItem>
-</Tabs>
+  - Any Teleport Agent whose invite token is defined in the Auth Service's
```

Comment:
Is this true? 

Agents that already have host certificates shouldn't need a join token, right?